src/main/java/org/stellar/sdk/SorobanServer.java

```diff
@@ -539,4 +560,12 @@ public enum Durability {
     TEMPORARY,
     PERSISTENT
   }
+
+  /** Describes additional resource leeways for transaction simulation. */
```

Comment:
for class organization, can this class be defined statically in SimulateTransactionRequest instead?

Reply:
Of course, I have made the modifications.